approval_frame/settings.py

```diff
@@ -9,7 +9,10 @@
 SECRET_KEY = 'jndet@)ed9ge4+!k605hc+$0tjb8u5oe)pdjab)-b3lsot_xnf'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+# For development purposes, set the DEBUG_* variables to True in a 
+# local_settings.py file, which we .gitignore. This file should be added
+# in the same directory that contains this settings.py file.
+DEBUG = False
 TEMPLATE_DEBUG = DEBUG
```

Comment:
Oh cool,  I didn't realize TEMPLATE_DEBUG was deprecated in 1.8. Need to check the differences between the versions.
Also, Since I have kept DEBUG=False in the settings.py file, I was required to add a 'localhost' to the ALLOWED_HOSTS list for the project to work out of the box. Perhaps it would need to be set to the sitename on production.